pyproject.toml

```diff
@@ -23,6 +23,8 @@ keywords = [
 ]
 requires-python = ">=3.8"
 dependencies = [
+  "ipywidgets",
+  "ipysheet",
```

Comment:
I guess we could seperate these out under a "notebooks" group? Maybe something for later...

Reply:
If the Notebook here is no longer going to be used I should remove these and leave them solely as dependencies of [`mesnage-org/pgfinder-jupyterlite`](https://github.com/mesnage-org/pgfinder-jupyterlite).